tests/functional.js

```diff
@@ -10,6 +10,7 @@ module.exports = [
   'tests/functional/sign_in.js',
   'tests/functional/sign_in_cached.js',
   'tests/functional/sign_in_blocked.js',
+  'tests/functional/sign_in_totp.js',
```

Comment:
@vbudhram should we add this to circle ci tests as well?

Reply:
Added!